lerna.json

```diff
@@ -4,7 +4,8 @@
     "framework",
     "platforms/**",
     "integrations/**",
-    "examples/**"
+    "examples/**",
+    "../botify/chatbots/team/botify-team-bot/"
```

Comment:
This was accidentally added, please remove for the PR.

Reply:
Done